cpp/include/Ice/Connection.h

```diff
@@ -54,54 +46,6 @@ namespace Ice
 
 namespace Ice
```

Comment:
The namespace Ice is declared twice in this file, we can merge the declarations.

Reply:
Fixed.